main.go

```diff
@@ -188,6 +188,7 @@ func main() {
 	flag.StringVar(&opts.CPUProfile, "cpuprofile", "", "write cpu profile to `file`")
 	flag.StringVar(&opts.MemProfile, "memprofile", "", "write memory profile to `file`")
 	flag.StringVar(&opts.MetricsEndpoint, "metrics-endpoint", cfg.Listen.MetricsEndpoint, "URL path at which to serve metrics")
+	flag.BoolVar(&opts.VerifyConfig, "verify-config", false, "Enable this flag to check config file loads, then exit")
```

Comment:
Can we also add documentation for this feature somewhere in the README? Maybe directly under "Usage", or alternatively in its own section under "Advanced features"?